frontend/src/lib/lemon-ui/icons/icons.tsx

```diff
@@ -990,6 +990,94 @@ export function IconInternetExplorer(props: LemonIconProps): JSX.Element {
     )
 }
 
+export function IconBlackberry(props: LemonIconProps): JSX.Element {
```

Comment:
Where did these come from? Ideally, we'd move them into @posthog/icons

Reply:
I drew them on figma and exported but will add them to the batch requests with the others :)